docs/release-notes.rst

```diff
@@ -1,6 +1,12 @@
 Release Notes
 =============
 
+Unreleased
+----------
+* New: Support for Django 3.0 and 3.1.
```

Comment:
Also update the supported versions in the readme?

Reply:
Good idea!